### PR TITLE
Initial setup

### LINKED ISSUE #48: D1.2: Setup initial backend project

### PR DESCRIPTION
**Pull request by solodezaldivar**
Setup of application with docker
Closes #48  #46

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      SPRING_DATASOURCE_NAME: icu-connect
+      SPRING_DATASOURCE_URL: jdbc:mysql://mysqldb:3306/icu-connect?createDatabaseIfNotExist=true&autoReconnect=true
+    depends_on:
+      - mysqldb
+    ports:
+      - 8080:8080
+    restart: always
+
+volumes:
+  mysqldb:
```
Comment (msalze) [inline docker-compose.yml:36]: Should this contain a value?

**Review by msalze — COMMENTED** [review]
Very good setup. Great to start working with.

File: docker-compose.yml (hunk 1 above)
Comment (solodezaldivar) [reply docker-compose.yml:36]: I don't think so, from what I gather this is standard procedure unless the volume is external to the docker-compose file as shown here https://docs.docker.com/storage/volumes/
image

Comment (msalze) [reply docker-compose.yml:36]: Great, thanks!

**Review by msalze — APPROVED** [review]

**Pull request merged**